ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Pipelines.resource

```diff
@@ -34,6 +35,30 @@ Create Pipeline Server
     Wait Until Generic Modal Disappears
     Wait Until Project Is Open    project_title=${project_title}
     ...    timeout-pre-spinner=5s    timeout-spinner=60s
+    # Check if there is any CA Bundle
+    # I did not find a better way to check if multi-line string is empty or "NONE"
+    ${length}=    Get Length    "${aws_s3_ca_bundle}"
+    IF    ${length} <= 6
+        ${aws_s3_ca_bundle}=    Get Variable Value    ${S3.AWS_CA_BUNDLE}    ${NONE}
+        ${length}=    Get Length    "${aws_s3_ca_bundle}"
+    END
+    IF    ${length} > 6
+        Patch Pipeline Server With CA Bundle
+        ...    project_title=${project_title}    aws_s3_ca_bundle=${aws_s3_ca_bundle}
+    END
+
+Patch Pipeline Server With CA Bundle
+    [Documentation]    Patch the underlying dspa resource with CA Bundle
+    [Arguments]    ${project_title}    ${aws_s3_ca_bundle}
+    ${dc_namespace}=    Get Openshift Namespace From Data Science Project    ${project_title}
+    ${rc}    ${out}=    Run And Return Rc And Output
+    ...    oc create configmap custom-ca-bundle -n ${dc_namespace} --from-literal=ca-bundle.crt='${aws_s3_ca_bundle}'    # robocop: disable
+    Should Be Equal As Integers    ${rc}    0
+    Log To Console    "${out}"
+    ${rc}    ${out}=    Run And Return Rc And Output
+    ...    oc patch dspa pipelines-definition -n ${dc_namespace} -p '[{"op":"add","path":"/spec/apiServer/cABundle","value":{"configMapKey":"ca-bundle.crt","configMapName":"custom-ca-bundle"}}]' --type json     # robocop: disable
+    Should Be Equal As Integers    ${rc}    0
+    Log To Console    "${out}"
```

Comment:
I think it can be only `Log`

Reply:
Is there anything wrong with printing it to the console log?

Reply:
With `Log`, you can see the content in the HTML report without going to the `.log` file.
Imho, it is easy.

Reply:
Ok, I will use the Log only